composer.json

```diff
@@ -59,7 +59,7 @@
         "pelago/emogrifier": "^v5.0.1",
         "phpoffice/phpspreadsheet": "^1.19",
         "prestashop/autoload": "^1.0",
-        "prestashop/blockreassurance": "^5.1",
+        "prestashop/blockreassurance": "^6",
```

Comment:
There was a problem with blockreassurance v6.0.0

There is no release on its repository https://github.com/PrestaShop/blockreassurance/releases

Reply:
Ok good to know, because it was included in the alpha...